ghdocs/BUILDING.md

```diff
@@ -21,37 +21,27 @@ The built files will be in the `/dist/` folder.
 
 ##Gulp tasks
 
-You can test and work with Fabric locally in a number of ways. 
+You can test and work with Fabric locally in using any of the tasks listed below. 
```

Comment:
Maybe:
You can test and work with Fabric locally, using any of the tasks listed below.